### PR TITLE
server: fix load leak

### PR DESCRIPTION
**Pull request by cfzjywxk**
### What problem does this PR solve? 

The `StartStopWatcher` goroutine should quit after load data processing

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
Execute load statements, and check goroutine stack

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

Release note

### CODE REVIEW

**Comment by cfzjywxk** [comment]
@imtbkcat @jackysp  PTAL

**Comment by imtbkcat** [comment]
@cfzjywxk does this pr need to be cherry-pick to release-3.0?

**Comment by cfzjywxk** [comment]
> @cfzjywxk does this pr need to be cherry-pick to release-3.0?

This change is on 4.0, 3.0 does all work in a single goroutine

File: server/conn.go [hunk 1]
```diff
@@ -1124,6 +1124,10 @@ func (cc *clientConn) handleLoadData(ctx context.Context, loadDataInfo *executor
 	loadDataInfo.InitQueues()
 	loadDataInfo.SetMaxRowsInBatch(uint64(loadDataInfo.Ctx.GetSessionVars().DMLBatchSize))
 	loadDataInfo.StartStopWatcher()
+	defer func() {
+		// let stop watcher goroutine quit
+		loadDataInfo.ForceQuit()
```
Comment (zimulala) [inline server/conn.go]:
```suggestion
		defer loadDataInfo.ForceQuit()
```

**Pull request merged**